CHANGES/1045.bugfix.rst

```diff
@@ -0,0 +1 @@
+Fixed ``__contains__`` throwing an exception for non strings.
```

Comment:
Not sure how to best rephrase it. I think it might give people more context if this was saying something like "There was this specific exception raise via `in` checks, and it is no longer happening". Or "The `in` check used to raise exception X when non-string value presence was being checked. It is now fixed to return false."
Ideas?